src/TransloaditClient.js

```diff
@@ -2,14 +2,14 @@ const reqr = global.GENTLY ? GENTLY.hijack(require) : require
 const got = reqr('got')
 const FormData = require('form-data')
 const crypto = reqr('crypto')
-const _ = reqr('underscore')
+const { isObject, isArray, extend, sumBy } = reqr('lodash')
```

Comment:
If I'm not mistaken this first loads all of loadsh, and then only uses 4 functions. If you require them separately like `const _isObject = require('lodash/isObject')` it should be more lightweight

Reply:
Indeed, I'll change that